playbooks/ansible.cfg

```diff
@@ -11,5 +11,17 @@ forks = 15
 # SSH timeout
 timeout = 120
 
+# Set the path to the folder in os-ansible-deployment which holds the dynamic
+# inventory script
+inventory = /opt/os-ansible-deployment/playbooks/inventory/
```

Comment:
We just need to add the "hostfile" setting here (its above in that file) and make a note that if using <1.9 for ansible you need to set that instead of inventory, and for >1.9 set inventory value. (And set hostfile also) Otherwise this works pretty well and definitely LGTM